Lesson-7/What do we offer.css

```diff
@@ -0,0 +1,244 @@
+@font-face {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 500;
+    src: url('../fonts/roboto-v27-latin-500.eot'); /* IE9 Compat Modes */
+    src: local(''),
+         url('../fonts/roboto-v27-latin-500.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+         url('../fonts/roboto-v27-latin-500.woff2') format('woff2'), /* Super Modern Browsers */
+         url('../fonts/roboto-v27-latin-500.woff') format('woff'), /* Modern Browsers */
+         url('../fonts/roboto-v27-latin-500.ttf') format('truetype'), /* Safari, Android, iOS */
+         url('../fonts/roboto-v27-latin-500.svg#Roboto') format('svg'); /* Legacy iOS */
+  }
+@media ( max-width: 2560px ) {
+  
+.heading {
+font-family: 'Roboto';
+font-style: normal;
+font-weight: 500;
+font-size: 40px;
+line-height: 47px;
+width: 306px;
+height: 47px;
+margin: 120px 567px 20px;
+
+}
+.container {
+    display: flex;
+    justify-content: space-between;
+    width: 1440px;
+}
+hr {
+    margin: 0px 670px 88px 670px;
+    width: 100px;
+    height: 4px;
+    background: #F5BD41;
+}
+div {
+    width: 296px;
+
+}
```

Comment:
are you sure that you want to add this style for all div? use classname for this goals